index.html

```diff
@@ -291,6 +312,59 @@
           </dt>
         </dl>
       </section>
+      <section>
+	<h2>
+	  Media Devices
+	</h2>
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.camera'>camera</dfn></code>,
+	  <code><dfn id='idl-def-PermissionName.microphone'>microphone</dfn>
+	  </code>,
+	  and <code><dfn id='idl-def-PermissionName.speaker'>speaker</dfn>
+	  </code>
+	  permissions are associated with permission to use media
+	  devices as specified in [[mediacapture-main]].
+	</p>
+	<p>These three permissions have an associated PermissionDescriptor:
+	</p>
+	<dl class='idl' title='dictionary DevicePermissionDescriptor :
+	    PermissionDescriptor'>
+	  <dt>
+	    DOMString deviceId
+	  </dt>
+	</dl>
+	<p>A permission covers access to the device given in the
+	associated descriptor.
+	</p>
+	<p>
+	  The permissions <code><dfn id='idl-def-PermissionName.all-cameras'>all-cameras</dfn></code>, <code><dfn id='idl-def-PermissionName.all-microphones'>all-microphones</dfn></code>,
+	  and <code><dfn id='idl-def-PermissionName.all-speakers'>all-speakers</dfn></code> 
+	cover access to all devices of that class, whether they exist
+	at the moment or are attached at a later time. When the "all"
+	  permissions have been granted, queries for permission to any
+	  specific device MUST return "granted" as long as the
+	  deviceId is a legal deviceId.
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.device-info'>device-info</dfn></code>
+	  permission controls access to names and capabilities of input
+	  and output devices.
+	</p>
+	<p>
+	  The permission storage identifier for permissions in this
```

Comment:
Trying to parse this. In https://github.com/w3c/mediacapture-main/issues/267 it seems we're zooming in to keying by origin and top-level origin.
Here we we say "origin and embedder's origin". Assuming we have more than one level of iFrame's, this would mean a difference between the two(?).